Source/CarthageKit/LocalDependencyStore.swift

```diff
@@ -0,0 +1,237 @@
+// swiftlint:disable vertical_parameter_alignment_on_call
```

Comment:
We shouldn't be disabling swiftlint rules. They're there for a reason: fix the violations.

Reply:
Removed the swiftlint rule disable. However, are you aware that there are many instances of this throughout the code base?